### PR TITLE
refactor: Improve UX -style sidebar (add room, rooms & search)with bootstrap

### LINKED ISSUE #103: style leftbar with bootstrap
style leftbar with bootstrap
tasks:

- style the search form with bootstrap's `form-control` class

- style the cards containing room name and description with bootstrap's `card` class under `card-columns` class of bootstrap.

- move the `add-room`(plus) button as a card under room list, so that user can click on a card to create new room.

Final look can be something like
baat-cheet
:

### PR DESCRIPTION
**Pull request by dvshsahu**
change style sidebar of the app with bootstrap.

Made following changes

- style the search bar with bootstrap form-control class, files index.ejs
- style rooms section with cards, introduced 'card-columns' container to hold room details,
- each room has room name and description.
- move the add button at the bottom after the room cards, style with the card.
- refactor common code for appending room and content to the method and called from multiple places.

fixes: #103

### CODE REVIEW

**Comment by LalitNM** [comment]
There are few more things you should work upon.

- Style the leftbar-toggle button with bootstrap.

- There is a lot of unnecessary style in style.css which corresponds to `.left` or `. person` or some other classes used in leftbar, you need to remove this code from there because we don't want any unnecessary code there.

- You can also remove any custom styling of search bar at the top of leftbar(leave it as a bootstrap's default `form-control`.

Do these and then I will review your PR again.

**Comment by dvshsahu** [comment]
Sure, I will be working on that, however need help on first point @LalitNM , 
- style the leftbar - toggle button with bootstrap : Here which button are we talking about, is it the add room button?

**Comment by LalitNM** [comment]
No, the button referred here is the button which appears on small screens. check this screenshot.
image

**Comment by dvshsahu** [comment]
okay sure

File: public/style.css [hunk 1]
```diff
@@ -88,17 +90,15 @@ body {
...
 .app-container .left .top:after {
   position: absolute;
   bottom: 0;
   left: 50%;
   display: block;
-  width: 80%;
+  width: 90%;
   height: 1px;
   content: '';
   background-color: #e6e6e6;
```
Comment (LalitNM) [inline public/style.css]: @dvshsahu can you think of something that we can get rid of this code also, perhaps a `container-fluid` can work(haven't tried it yet).

**Review by LalitNM — CHANGES_REQUESTED** [review]
Everything else looks great except a small bug of content overflow on small screen. Screen-shot: 
Screen Shot 2020-10-09 at 19 04 24

Don't you think that we should put a scrollbar here?
And can you also increase the size of sidebar-btn?

**Comment by dvshsahu** [comment]
Yes, scrollbar is needed there for both screens but I was under impression that separate issue will be there. No worries, I will take it up now.
Also, for button size, I feel this size is decent enough if the screen is too small, I will share screenshots with bigger sizes, we can discuss and fix this as well.

**Comment by LalitNM** [comment]
@dvshsahu if you think that making a scrollbar is too much for one PR, then leave it. I will make another issue after merging this. And for the sidebar-btn I also think a little bit increase in font-size would be enough. But try to fix the changes in .top class. Then I will merge this.

**Comment by dvshsahu** [comment]
I have already made some progress for scroll, I would like to complete it, is that fine @LalitNM 
For font size, sure thing I will go ahead and make the update.

File: public/style.css (hunk 1 above)
Comment (dvshsahu) [reply public/style.css]: removed the code, now using plain <hr> tag to achieve the same

**Review by LalitNM — APPROVED** [review]

**Comment by LalitNM** [comment]
I merged:rocket: this @dvshsahu after some minor changes from my side. Hope you enjoyed working on this projest.

**Comment by dvshsahu** [comment]
Thank you so much, it was a good learning experience. Hope to connect on this project again.

**Pull request merged**